How-To-Setup-Internal-Package-Repository.md

```diff
@@ -325,17 +308,19 @@ Note the section above where you should insert the code to test your packages be
 
 ```powershell
   function ConvertTo-ChocoObject {
+
       [CmdletBinding()]
+
       Param (
           [Parameter(ValueFromPipeline)]
           [string]$InputObject
       )
 
       Process {
-          # format of the 'choco list' output is:
+          # format of the 'choco list -r' output is:
           # <PACKAGE NAME> <VERSION> (ie. adobereader 2015.6.7)

```

Comment:
After adding the `-r` parameter the format of the output changes. Can you amend this comment to reflect that?

Reply:
Addressed